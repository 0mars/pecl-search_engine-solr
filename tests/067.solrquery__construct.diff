001+ Fatal error: Uncaught exception 'SolrIllegalArgumentException' with message 'SolrQuery::setQuery() expects parameter 1 to be string, object given' in /home/grim/c-workspace/php-src/ext/solr/tests/067.solrquery__construct.php:7
001- 
002+ Stack trace:
003+ #0 /home/grim/c-workspace/php-src/ext/solr/tests/067.solrquery__construct.php(7): SolrQuery->setQuery(Object(stdClass))
004+ #1 {main}
005+   thrown in /home/grim/c-workspace/php-src/ext/solr/tests/067.solrquery__construct.php on line 7