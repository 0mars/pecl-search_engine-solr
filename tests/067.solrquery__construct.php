<?php

//$doc = new SolrInputDocument();

$query = new SolrQuery();

$query->setQuery(new StdClass());

?>
