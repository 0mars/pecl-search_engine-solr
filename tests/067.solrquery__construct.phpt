--TEST--
SolrQuery::_construct - Throws exception on illegal Argument
--FILE--
<?php

$query = new SolrQuery(new StdClass());

?>
--EXPECTF--