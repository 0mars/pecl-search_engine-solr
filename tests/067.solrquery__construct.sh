#!/bin/sh

/home/grim/c-workspace/php-src/installed/php-55-debug-zts/bin/php  -n -c '/home/grim/c-workspace/php-src/ext/solr/tmp-php.ini'  -d "output_handler=" -d "open_basedir=" -d "safe_mode=0" -d "disable_functions=" -d "output_buffering=Off" -d "error_reporting=32767" -d "display_errors=1" -d "display_startup_errors=1" -d "log_errors=0" -d "html_errors=0" -d "track_errors=1" -d "report_memleaks=1" -d "report_zend_debug=0" -d "docref_root=" -d "docref_ext=.html" -d "error_prepend_string=" -d "error_append_string=" -d "auto_prepend_file=" -d "auto_append_file=" -d "magic_quotes_runtime=0" -d "ignore_repeated_errors=0" -d "precision=14" -d "memory_limit=128M" -d "opcache.fast_shutdown=0" -d "opcache.file_update_protection=0" -d "extension_dir=/home/grim/c-workspace/php-src/ext/solr/modules/" -d "extension=solr.so" -d "session.auto_start=0" -f "/home/grim/c-workspace/php-src/ext/solr/tests/067.solrquery__construct.php"  2>&1
