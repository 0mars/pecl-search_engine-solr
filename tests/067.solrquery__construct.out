Fatal error: Uncaught exception 'SolrIllegalArgumentException' with message 'SolrQuery::setQuery() expects parameter 1 to be string, object given' in /home/grim/c-workspace/php-src/ext/solr/tests/067.solrquery__construct.php:7
Stack trace:
#0 /home/grim/c-workspace/php-src/ext/solr/tests/067.solrquery__construct.php(7): SolrQuery->setQuery(Object(stdClass))
#1 {main}
  thrown in /home/grim/c-workspace/php-src/ext/solr/tests/067.solrquery__construct.php on line 7